app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,8 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :credentials,                  :class_name => "ManageIQ::Providers::AutomationManager::Authentication",
```

Comment:
because it's different. 

[here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_tower/automation_manager.rb#L9) we delegate `:authentications` to the provider. Because we expect `ems.authentications` to be those that are 1) in @blomquisg list.

This association here is for 2) in his list.

Thats why I'm arguing for a rename